### ISSUE TITLE
Melbourne (MEL) airport icon does not show up on map

### ISSUE THREAD

**Issue by eng3** [jpatokal/openflights#888]
When viewing the map, it will show my route to/from MEL (Melbourne Australia), however the circular icon is not displayed

**Comment by jpatokal** [comment]
Have you tried scrolling the map to the right?  Due to issues with OpenLayers, it's currently not possible to show airport icons across the date line.

**Comment by eng3** [comment]
Yes, it is not the date line issue.  For example SYD displays the icon.  The routing displays ok, just not MEL.  See attached
![mel1](https://user-images.githubusercontent.com/21344196/45586837-19788200-b8cb-11e8-911f-a4a0bbaf3fda.jpg)
![mel2](https://user-images.githubusercontent.com/21344196/45586838-19788200-b8cb-11e8-8b27-ff8b1802d76b.jpg)

**Comment by eng3** [comment]
Also, I see a comment on the dateline issue from 2012 with a suggested solution to that issue.
